packages/react-components/src/components/Picker/Picker.module.scss

```
@@ -33,4 +34,9 @@ $base-class: 'picker';
       margin-right: 5px;
     }
   }
+
+  &__tag {
+    margin-bottom: 4px;
+    margin-right: 4px;
```

How about this? Or we want to avoid declaring 0 margin for remaining directions?
```suggestion
    margin: 0 4px 4px 0;
```